tests/default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import <nixpkgs> {} }:
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
zhy the refor,qtting of this file ?

Reply:
I think what @teto was trying to say was "why the reformatting of this file?" 😉

Basically, avoid running nixfmt manually, simply run the format script in the project root.

Reply:
This file is still having an undesired major reformatting. The only change that should be here is the addition of the `./modules/programs/rofi-pass` line.